### PR TITLE
git lecture added

### PR DESCRIPTION
**Pull request by UtkarshMishra04**

### CODE REVIEW

File: _posts/2018-10-09-Git-Lecture.md [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+
+	- #### Usage
+	```bash
+	$ git branch --list # display all local branches
+	$ git branch # same as above
+	$ git branch -r # display all remote branches
+	$ git branch new-branch # create a new branch
+	$ git branch -d old-branch # Delete a branch if the local old-branch isn't ahead of origin/old-branch in commits. Shortcut for --delete.
+	$ git branch -D old-branch # force delete a branch anyways. Shortcut for --delete --force
+	```
```
[the remark this answers is not in the document]
Comment (dakshitagrawal97) [reply _posts/2018-10-09-Git-Lecture.md]: `git push remote --delete <branch-name>`

I think this is the command.

File: _posts/2018-10-09-Git-Lecture.md [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+```
+
+## Configure username and email
+```bash
+$ git config --global user.name 'RohanBh'
+$ git config --global user.email 'brohan52@gmail.com’
+```
+
+## Install Atom or Sublime Text Editor
+### Atom
```
[the remark this answers is not in the document]
Comment (dakshitagrawal97) [reply _posts/2018-10-09-Git-Lecture.md]: Not aware of it.  Probably add `vsscode-oss` as an additional editor.

[the remark this answers is not in the document]
Comment (dakshitagrawal97) [reply _posts/2018-10-09-Git-Lecture.md]: Not aware of it.  Probably add `vsscode-oss` as an additional editor.

File: _posts/2018-10-09-Git-Lecture.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dakshitagrawal97) [reply _posts/2018-10-09-Git-Lecture.md]: `git push remote --delete <branch-name>`

I think this is the command.

[the remark this answers is not in the document]
Comment (dakshitagrawal97) [reply _posts/2018-10-09-Git-Lecture.md]: `git push remote --delete <branch-name>`

I think this is the command.

[the remark this answers is not in the document]
Comment (dakshitagrawal97) [reply _posts/2018-10-09-Git-Lecture.md]: `git push remote --delete <branch-name>`

I think this is the command.

File: _posts/2018-10-09-Git-Lecture.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (dakshitagrawal97) [reply _posts/2018-10-09-Git-Lecture.md]: These are the standard editors used.  We can probably add `vscode-oss` as an additional text editor?

File: _posts/2018-10-09-Git-Lecture.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dakshitagrawal97) [reply _posts/2018-10-09-Git-Lecture.md]: `git push remote --delete <branch-name>`

I think this is the command.

File: _posts/2018-10-09-Git-Lecture.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (dakshitagrawal97) [reply _posts/2018-10-09-Git-Lecture.md]: These are the standard editors used. We can probably add `vscode-oss` as an additional text editor?

File: _posts/2018-10-09-Git-Lecture.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (r1walz) [reply _posts/2018-10-09-Git-Lecture.md]: both are correct `git push <remote> :<branch>` is just the short-form of it.

**Pull request merged**